### DISCUSSION TITLE
Lesson 2 (Intro to Data Ethics) Post Lecture Quiz [Spoilers]

### DISCUSSION THREAD

**Discussion by irvinktang** [microsoft/Data-Science-For-Beginners#329] [General]
Hi,

On the last question of the post lecture quiz for lesson 2, I'm wondering why "collection bias" is included as an answer to "which of these is a real ethics principle and practice that can be applied?". Is collection bias not an ethics challenge?

**Comment by drjaber2000** [comment]
hello 
in my humble opinion 
(collection bias) is a practice based on the  principle of fairness where data extracted from different sub-population  unfairly
